### ISSUE TITLE
Own ExecCommand does not find executable if not run via phar / why is __DIR__ used in findProjectBin

### ISSUE THREAD

**Issue by amenk** [consolidation/Robo#613]
### Steps to reproduce
* Extend ExecCommand as in  
https://github.com/iMi-digital/robo-typo3/blob/master/src/Task/Typo3/Stack.php
* RoboFile.php in typo3project contains `public testcommand() { $this->_typo3cms('database:export } `
* Execute via 
     cd typo3project; /path/to/robo/source/robo testcommand
* typo3console is in typo3project/vendor/bin/typo3cms

### Expected behavior

Command is executed

### Actual behavior

     [error]    in task iMi\RoboTypo3\Task\Typo3\Stack 

     typo3cms installation could be found. 

### System Configuration
Ubuntu 16.04, php7

When I build the phar where I include the commands, it works. But not when I run robo directly (which I do during development of new commands).

Basically it might be sufficient to add more comments about the intents of `\Robo\Common\ExecCommand::findProjectBin` - I do not understand why __DIR__ is used.

**Comment by amenk** [comment]
Should findProjectBin  find the ROBO project bin or the folder where the RoboFile is in?

**Comment by greg-1-anderson** [comment]
`vendor/bin` from the cwd should be in the search path for findProjectBin. A PR to fix this would be welcome if it is not working as intended.
